golang/cmd/sensorconnect/main.go

```diff
@@ -133,7 +134,7 @@ func main() {
 		if subTwentyMs {
 			zap.S().Warnf("Going under 20MS is not recommendet with IFM IO-Link Masters")
 		} else {
-			panic("LOWER_POLLING_TIME under 20 can IFM IO-Link Master failures, set SUB_TWENTY_MS to 1 to continue !")
+			zap.S().Fatal(err)
```

Comment:
really want to remove the concrete error message here?

Reply:
No, those where regex errors

Reply:
maybe push your changes? at the moment these are not active yet